scripts/skills/perks/perk_rf_tempo.nut

```diff
@@ -95,10 +96,11 @@ this.perk_rf_tempo <- ::inherit("scripts/skills/skill", {
 
 	function onBeforeTargetHit( _skill, _targetEntity, _hitInfo )
 	{
-		if (this.m.APRecovery.len() != 0 && _skill.isAttack() && ::Tactical.TurnSequenceBar.isActiveEntity(this.getContainer().getActor()) && !_targetEntity.isTurnDone() && !_targetEntity.isTurnStarted())
+		if (this.m.AttacksRemaining > 0 && _skill.isAttack() && ::Tactical.TurnSequenceBar.isActiveEntity(this.getContainer().getActor()) && !_targetEntity.isTurnDone() && !_targetEntity.isTurnStarted())
 		{
+			this.m.AttacksRemaining--;
```

Comment:
If I hit a target, whose turn is already done or whose turn has already startet, then none of my two AttacksRemaining are taken away from me.
The conditions `!_targetEntity.isTurnDone() && !_targetEntity.isTurnStarted()` must be put in an extra `if` after this line, in order to prevent that.